src/mlpack/tests/CMakeLists.txt

```
@@ -175,12 +175,21 @@ add_executable(mlpack_test
   main_tests/test_helper.hpp
 )
 
-# Link dependencies of test executable.
-target_link_libraries(mlpack_test
-  mlpack
-  ${ARMADILLO_LIBRARIES}
-  ${COMPILER_SUPPORT_LIBRARIES}
+if(NOT BUILD_SHARED_LIBS)
+# Link dependencies statically of test executable, when cross compiling.
+  target_link_libraries(mlpack_test -static
+    mlpack
+    ${ARMADILLO_LIBRARIES}
+    ${COMPILER_SUPPORT_LIBRARIES}
 )
```

```suggestion
  )
```
This poor parentheses didn't get indented. :smile: